packages/react/src/UnorderedList/UnorderedList.tsx

```diff
@@ -13,13 +13,21 @@ interface UnorderedListComponent
   Item: typeof UnorderedListItem
 }
 
+export interface UnorderedListProps extends HTMLAttributes<HTMLUListElement> {
+  noMarker?: boolean
+}
+
 export const UnorderedList = forwardRef(
   (
-    { children, className, ...restProps }: PropsWithChildren<HTMLAttributes<HTMLUListElement>>,
+    { children, noMarker = false, className, ...restProps }: PropsWithChildren<UnorderedListProps>,
```

Comment:
I thought we opted for `noMarkers`? We would have to change this in the classes and story names as well

Reply:
Yes, I’ll update this.